_commands/basics/pwd.md

```diff
@@ -0,0 +1,49 @@
+---
+---
+
+pwd
+--
+
+`pwd` stands for print working directory. `pwd` outputs the full pathname of the current work director. 
+
+~~~ bash
+$ pwd
+/home/bo/Desktop
+~~~
+
+`pwd` can also be used to store the full path to the current directory. 
+
+~~~ bash
+x=$(pwd)
+~~~
+
+### Useful Options / Examples 
+
+#### `pwd -L`
+
+~~~ bash
+$ ln -s . test
+$ cd test && pwd
+/home/bo/Desktop/test
+$ /bin/pwd
+/home/bo/Desktop
+$ /bin/pwd -L
+/home/bo/Desktop/test
+~~~
+
+##### Break it down
+* `--logical` If the contents of the environment variable 'PWD' provide an absolute name of the current directory with no '.' or '..' components, but possibly with symbolic links, then output those contents. Otherwise, fall back to default -P handling. 
```

Comment:
This reads very densely, like a man page. We're trying to be a little more accessible. Can you expand on this text a little to make it easier to understand?